### ISSUE TITLE
Problem running command export: Forbidden [v7.12]

### ISSUE THREAD

**Issue by reuben-pct** [auth0/auth0-deploy-cli#537]
### Description

Since upgrading to `auth0-deploy-cli v7.12`, running the deploy command fails. After rolling back to `7.11.1`, it succeeds.

### Reproduction

This is the failing output when running the output to yaml format
```
a0deploy export --config_file config.json --format yaml --output_folder .
```

> error: Problem running command export
> error: Problem loading tenant data from Auth0 Forbidden: The account is not allowed to perform this operation, please contact our support team

This is the failing output when running the output to directory format
```
a0deploy export --config_file config.json --format directory --output_folder .
```

> error: Problem running command export during stage load when processing type customDomains
> error: The account is not allowed to perform this operation, please contact our support team

### Environment

My `auth0-deploy-cli` installed locally in the directory (i avoid global installs where possible and run via `yarn run x` to run the locally installed lib.

This was happening on a fresh install / account. I have only just created the auth0 account and tenant a couple days ago and this was the first time using the deploy cli tool to export the data to version track it.

I use `yarn` as my paclage manager and I'm running on a Macbook Pro M1 chip MacOS Monterey 12.3.1.

As mentioned, v 7.12 of the tool has this issue, but after rolling back to 7.11.1 it works happily.

**Comment by ezefdeil** [comment]
Same here, rolling back to 7.11.1 version works like a charm

**Comment by willvedd** [comment]
This has been fixed in the [7.12.2](https://github.com/auth0/auth0-deploy-cli/releases/tag/v7.12.2) release. As discovered above, custom domains are not available to all tenants and subscriptions. Most notably they are not supported for free-tiered tenants. By detecting this specific scenario, we are able to effectively skip custom domains. Please update and give it a shot. 

Thanks for opening the ticket!
